src/main/java/stream/flarebot/flarebot/commands/secret/UpdateCommand.java

```diff
@@ -63,7 +64,7 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                                 public void run() {
                                     update(true, channel);
                                 }
-                            }.delay(p.toStandardSeconds().getSeconds() * 1000);
+                            }.delay(TimeUnit.SECONDS.toMillis(p.toStandardSeconds().getSeconds()));
```

Comment:
I wouldn't bother doing it here it says seconds and we times 1000 it's quite obvious